### ISSUE TITLE
Unable to use vscode as the editor in windows 10

### ISSUE THREAD

**Issue by maple3142** [Tampermonkey/tamperdav#1]
`open-in-editor` is set to `code` in `config.json`, but it open notepad++ when I click on the cloud icon.

`which code` result:
```text
C:\Users\Administrator>which code
C:\Program Files (x86)\Microsoft VS Code\bin\code
```

**Comment by Revadike** [comment]
Maybe develop a plugin for vscode?

**Comment by derjanb** [comment]
Please try to enter the full path (C:\Program Files (x86)\Microsoft VS Code\bin\code) at the "open-in-editor" option or include "C:\Program Files (x86)\Microsoft VS Code\bin\" in the default search path.

**Comment by maple3142** [comment]
`C:\Program Files (x86)\Microsoft VS Code\bin` has already been incuded in `PATH`.
And it doesn't work after I set `open-in-editor` to full path.
`config.json`:
```json
{
    "path": "C:\\Users\\Administrator\\Documents\\GitHub\\userscripts",
    "no-auth-warning": false,
    "username": "AAAAAA",
    "password": "BBBBBB",
    "port": "7878",
    "host": "localhost",
    "max-cursors": 512,
    "open-in-editor": "C:\\Program Files (x86)\\Microsoft VS Code\\bin\\code",
    "meta-touch": true,
    "debug": true
}
```

**Comment by maple3142** [comment]
Why set `editor` to `undefined` here?
I removed that line and custom editor could work as expected.
https://github.com/Tampermonkey/tamperdav/blob/2d610f098b96a5b499d6ea36cea8ed7f30e71eb5/server.js#L140-L142

Besides, [node-open](https://github.com/pwnall/node-open) have been unmaintained for years.
It has some problem on Windows when I test it, so I recommend use [opn](https://github.com/sindresorhus/opn) instead for better compatibility.

**Comment by derjanb** [comment]
> Why set editor to undefined here?

Good catch. :) Shoud be fixed by 69b9646.

> It has some problem on Windows when I test it, so I recommend use opn instead for better compatibility.

IIRC I've tried this, but there were issues, but I'm not sure. Fell free to file a pull request. :)
